ChangeLog.txt

```diff
@@ -32,6 +34,17 @@ MINOR:
 - Modified the birthdates of Claffo Lethings's descendants for more realistic intervals and childbirths
 - Prevented the ERE event about sending gold to the Ostrogths from firing if the two realms are at war
 - Restored the ambitions that were removed for those who have Conclave 
+- Fixed issue with Root West Pagans where only monkeys are named as gods
+- Antinomialins can now do what Paul says they did in the Epistles: INCEST!
+- Antinomialins now love to screw, as Paul noted in Epistles ( ͡° ͜ʖ ͡°)
+- Antinomialins party hard and party often, as Paul noted in Epistles  ( ͡° ͜ʖ ͡°)
```

Comment:
@loup99 

It is. The Antinomians were so bad that the Apostle Paul dedicated _entire chapters_ to refuting their twisted interpretation. What source is more authoritative than the person who addressed them directly as they existed? The tooltip _already says_ that they are free from all morality "by God's grace". All I did was implement that in-game.